{
    "config": {
        "abort": {
            "already_configured": "Il dispositivo \u00e8 gi\u00e0 configurato",
            "cannot_connect": "Impossibile connettersi",
            "unknown": "Errore imprevisto"
        },
        "error": {
            "cannot_connect": "Impossibile connettersi",
            "unknown": "Errore imprevisto"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Host"
                },
                "data_description": {
                    "host": "Il nome host o l'indirizzo IP del dispositivo Smarty"
                }
            }
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_auth_error": {
            "description": "La configurazione di {integration_title} tramite YAML \u00e8 stata rimossa, ma si \u00e8 verificato un errore di autenticazione durante l'importazione della configurazione esistente. \nL'installazione non verr\u00e0 eseguita. \n\nVerifica che {integration_title} funzioni correttamente e riavvia Home Assistant per tentare di nuovo l'importazione. \n\nIn alternativa, puoi rimuovere completamente la configurazione `{domain}` dal tuo configuration.yaml, riavviare Home Assistant e aggiungere manualmente l'integrazione {integration_title}.",
            "title": "Importazione YAML non riuscita a causa di un errore di autenticazione"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "La configurazione di {integration_title} tramite YAML \u00e8 stata rimossa, ma si \u00e8 verificato un errore sconosciuto durante l'importazione della configurazione esistente. \nL'installazione non verr\u00e0 eseguita. \n\nVerifica che {integration_title} funzioni correttamente e riavvia Home Assistant per tentare di nuovo l'importazione. \n\nIn alternativa, puoi rimuovere completamente la configurazione `{domain}` dal tuo configuration.yaml, riavviare Home Assistant e aggiungere manualmente l'integrazione {integration_title}.",
            "title": "Importazione YAML fallita con errore sconosciuto"
        }
    }
}